frontend/src/views/WheresMyGene/components/RightSideBar/style.ts

```diff
@@ -0,0 +1,32 @@
+import styled from "@emotion/styled";
+import {
+  SideBarPositioner as RawSideBarPositioner,
+  SIDEBAR_BOX_SHADOW_COLOR,
+} from "src/components/common/SideBar/style";
+import { HEADER_HEIGHT_PX } from "src/components/Header/style";
+
+export const RightSideBarPositioner = styled(RawSideBarPositioner)`
+  max-height: ${sectionMaxheight};
```

Comment:
calculating the max height of the right sidebar components for single and split cases